torch/_dynamo/eval_frame.py

```diff
@@ -918,10 +918,7 @@ def inner(*args, **kwargs):
         out_guards = None
         graph_captured_input = None
         graph_captured_result: Optional[Tuple[torch.Tensor, ...]] = None
-        fake_mode = _guards.detect_fake_mode(args)
-        _allow_fake_constant: bool = (
-            fake_mode is not None
-        )  # Allow fake constants during symbolic tracing
+        fake_mode = None
```

Comment:
Does this PR addresses https://github.com/pytorch/pytorch/issues/106412?

Reply:
Yes, it fixes that